compatibility_lib/compatibility_lib/configs.py

```diff
@@ -14,6 +14,11 @@
 
 """Common configs for compatibility_lib."""
 
+IGNORED_DEPENDENCIES = [
```

Comment:
Add a comment about why we ignore these dependencies would be nice.

Reply:
Done.